src/common/com/intellij/plugins/haxe/config/sdk/HaxeSdkType.java

```diff
@@ -117,7 +117,7 @@ public void saveAdditionalData(SdkAdditionalData additionalData, Element additio
   public FileChooserDescriptor getHomeChooserDescriptor() {
     final FileChooserDescriptor result = super.getHomeChooserDescriptor();
     if (SystemInfo.isMac) {
-      result.putUserData(PathChooserDialog.NATIVE_MAC_CHOOSER_SHOW_HIDDEN_FILES, Boolean.TRUE);
+      result.withShowHiddenFiles(true); // TODO: Test on a mac: converted for v15.
```

Comment:
on side note, are you going to follow-up on that TODO note you left there... having one of our engineers (who uses Mac) sign up for validating it